src/sagemaker/estimator.py

```diff
@@ -771,13 +773,17 @@ class Framework(EstimatorBase):
     MPI_NUM_PROCESSES_PER_HOST = 'sagemaker_mpi_num_of_processes_per_host'
     MPI_CUSTOM_MPI_OPTIONS = 'sagemaker_mpi_custom_mpi_options'
 
-    def __init__(self, entry_point, source_dir=None, hyperparameters=None, enable_cloudwatch_metrics=False,
-                 container_log_level=logging.INFO, code_location=None, image_name=None, dependencies=None, **kwargs):
+    def __init__(self, entry_point, git_config=None, source_dir=None, hyperparameters=None,
```

Comment:
Can you move `git_config` to the end of the constructor params? Other tests may init estimators without specifying optional param name, which may cause existing tests fail. Or you can inspect all tests that use this constructor and modify them correspondingly.